keycloak-event-emitter-tests/src/test/java/io/cloudtrust/keycloak/AbstractTest.java

```diff
@@ -102,4 +164,28 @@ public static WebArchive deploy() {
                 .addAsServiceProvider(EventEmitterProviderFactory.class);
     }
 
+    private void setupTestRealm() {
+        RealmRepresentation realm = testRealm.toRepresentation();
+        realm.getEventsListeners().add(EventEmitterProviderFactory.PROVIDER_ID);
+        testRealm.update(realm);
+
+        CredentialRepresentation credentialPass = new CredentialRepresentation();
+        credentialPass.setType(CredentialRepresentation.PASSWORD);
+        credentialPass.setValue("password");
+
+        {
+            UserRepresentation testUser = new UserRepresentation();
+            testUser.setUsername("test.user");
+            testUser.setEmail("test.user@test.com");
+            testUser.setEnabled(true);
+            if (testUser.getCredentials() == null) {
+                testUser.setCredentials(new LinkedList<>());
+            }
+            testUser.getCredentials().add(credentialPass);
+            Response createUser = testRealm.users().create(testUser);
```

Comment:
Response is AutoCloseable... you can use try (Response x = ...) { ... }

Reply:
fixed